project/Versions.scala

```diff
@@ -6,7 +6,7 @@ object Versions {
   val Paranamer       = "2.8"
   val ScalaLogging    = "3.5.0"
 
-  val DataStaxJavaDriver = "4.5.0"
+  val DataStaxJavaDriver = "4.7.2"
```

Comment:
Small note, mainly historical; if something less than 4.6.0 is used here another dependency (jnr-ffi) comes into play.  That lib was removed as an explicit dependency of the Java driver in 4.6.0; see JAVA-2662 for more details.